### PR TITLE
Post approver suggestion rather than notifying approvers

### PR DESCRIPTION
**Pull request by crimsonfaith91**
Addressing https://github.com/kubernetes/test-infra/issues/2366

### CODE REVIEW

File: mungegithub/mungers/approvers/owners.go [hunk 1]
```diff
@@ -464,7 +464,9 @@ func GenerateTemplateOrFail(templ, name string, data interface{}) *string {
 func GetMessage(ap Approvers, org, project string) *string {
 	message := GenerateTemplateOrFail(`This pull-request has been approved by: {{range $index, $approval := .ap.ListApprovals}}{{if $index}}, {{end}}{{$approval}}{{end}}
 {{- if not .ap.IsApproved}}
-We suggest the following additional approver{{if ne 1 (len .ap.GetCCs)}}s{{end}}: {{range $index, $cc := .ap.GetCCs}}{{if $index}}, {{end}}@{{$cc}}{{end}}
+We suggest the following additional approver{{if ne 1 (len .ap.GetCCs)}}s{{end}}: {{range $index, $cc := .ap.GetCCs}}{{if $index}}, {{end}}{{$cc}}{{end}}
+
+Assign the PR to him or her with a comment when ready.
```
Comment (cblecker) [inline mungegithub/mungers/approvers/owners.go]: Perhaps give an example with a code block:
```
/assign @alice
```

**Comment by thockin** [comment]
@k8s-bot ok to test

**Comment by thockin** [comment]
+1

File: mungegithub/mungers/approvers/owners.go (hunk 1 above)
Comment (apelisse) [inline mungegithub/mungers/approvers/owners.go]: I think you can use "them". Would work for plural and gender-neutral singular.

Comment (apelisse) [inline mungegithub/mungers/approvers/owners.go]: Yes, ideally, something that can be copy-pasted

File: mungegithub/mungers/approvers/owners.go [hunk 2]
```diff
@@ -464,7 +464,9 @@ func GenerateTemplateOrFail(templ, name string, data interface{}) *string {
 func GetMessage(ap Approvers, org, project string) *string {
 	message := GenerateTemplateOrFail(`This pull-request has been approved by: {{range $index, $approval := .ap.ListApprovals}}{{if $index}}, {{end}}{{$approval}}{{end}}
 {{- if not .ap.IsApproved}}
-We suggest the following additional approver{{if ne 1 (len .ap.GetCCs)}}s{{end}}: {{range $index, $cc := .ap.GetCCs}}{{if $index}}, {{end}}@{{$cc}}{{end}}
+We suggest the following additional approver{{if ne 1 (len .ap.GetCCs)}}s{{end}}: {{range $index, $cc := .ap.GetCCs}}{{if $index}}, {{end}}{{$cc}}{{end}}
```
Comment (apelisse) [inline mungegithub/mungers/approvers/owners.go]: Maybe I would make the names bold or italic to emphasize (that was done by the mention, now it's not going to stand-out).

File: mungegithub/mungers/approvers/owners.go (hunk 1 above)
Comment (crimsonfaith91) [inline mungegithub/mungers/approvers/owners.go]: Great idea! :)

File: mungegithub/mungers/approvers/owners.go (hunk 2 above)
Comment (crimsonfaith91) [inline mungegithub/mungers/approvers/owners.go]: I will get the names bold. :)

**Comment by crimsonfaith91** [comment]
I am verifying the template changes. Is there any PR that has the /approve message? Mind to point me to that?

File: mungegithub/mungers/approvers/owners.go (hunk 1 above)
Comment (crimsonfaith91) [inline mungegithub/mungers/approvers/owners.go]: Does code block means something that can be copy-pasted?

Comment (crimsonfaith91) [inline mungegithub/mungers/approvers/owners.go]: @apelisse Do you mind to point me to a PR that has a code block that can be copy-pasted? Thanks!

Comment (cblecker) [inline mungegithub/mungers/approvers/owners.go]: @crimsonfaith91 I think @apelisse is more suggesting that the text generated in the message be copy-pastable

Such as:
We suggest the following additional approver: cblecker

Assign the PR to them with a comment when ready:
`/assign @cblecker`

Comment (crimsonfaith91) [inline mungegithub/mungers/approvers/owners.go]: Got it! I was looking into zero-clipboard. Haha. Thanks!

**Comment by crimsonfaith91** [comment]
@cblecker thanks for approving. i submitted another commit to fix a formatting issue. :)

**Comment by cblecker** [comment]
One last thing: is it possible to squash the commits prior to merge?

**Comment by crimsonfaith91** [comment]
sure! i will do it now! :)

**Comment by crimsonfaith91** [comment]
done! currently waiting for build tests.

**Comment by cblecker** [comment]
All tests passing. LGTM.

PTAL, @apelisse :)

**Pull request merged**